### PR TITLE
[ALLUXIO-1960] Add Instance Group support to GCE deployments

### PR DESCRIPTION
**Pull request by ericmand**
Addresses: https://alluxio.atlassian.net/browse/ALLUXIO-1960

### CODE REVIEW

File: deploy/vagrant/conf/gce.yml.template [hunk 1]
```diff
@@ -64,6 +64,10 @@ Prefix:
 # See documentation (https://cloud.google.com/compute/docs/networking#addingafirewall) for more details.
 Network: "default"
 
+#The name of the Instance Group to which workers will be deployed. If empty, instance groups are not used.
```
Comment (uronce-cc) [inline deploy/vagrant/conf/gce.yml.template]: NIT:  add a space after `#`.

**Comment by uronce-cc** [comment]
@ericmand Thanks for this! Apart from a comment on formatting, LGTM. Also, could you sign the license agreement please?

**Comment by jsimsa** [comment]
@ericmand thank you for the PR ... could you please create a JIRA issue on alluxio.atlassian.net for this PR (as per instructions on http://alluxio.org/documentation/master/en/Contributing-to-Alluxio.html)

**Comment by ifcharming** [comment]
ok to test

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**